Source: MontaVista Software, Inc.
MR: 28237 
Type: Enhancement
Disposition: MontaVista
Signed-off-by: muvarov@ru.mvista.com
Description:
	1. make bootcycle load after udev.
	2. reopen /dev/initctl after udev starts.


diff -Nuarp bootcycle-1.0_orig/bootcycle bootcycle-1.0/bootcycle
--- bootcycle-1.0_orig/bootcycle	2008-05-12 13:15:52.000000000 +0400
+++ bootcycle-1.0/bootcycle	2008-05-12 14:31:40.000000000 +0400
@@ -66,6 +66,8 @@ BCD_REBOOT_COUNT=$(($BCD_REBOOT_COUNT+1)
 if [ $BCD_REBOOT_COUNT -gt $BCD_MAX_REBOOTS ]; then
     # shut down the box
     echo BCD_REBOOT_COUNT=0 > $STATUS_FILE
+    # make init reopen /dev/initctl
+    kill -s SIGUSR1 1
     $SHUTDOWN_CMD
     exit
 fi
